packages/config/docs/file_paths.md

```diff
@@ -0,0 +1,107 @@
+This document explains some of the main file paths used in a Netlify Site. This is mostly intended for code
```

Comment:
[sand] I'd expect "site" to be lowercase here, since it's probably a general site and not a specific proper noun Site you're talking about. In fact, I see capitalized "Sites" in a few different places in this documentation. Unless it's a specific feature name, you can make those references lowercase.

Reply:
I tend to capitalize it to distinguish a Netlify Site from a website, which is a slightly different concept, since Netlify Sites include all Netlify features with them. However, if we prefer lowercasing it, then we should be consistent, so I've just fixed it. Thanks for the feedback! :+1: